main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+Now you can install Docker!
+`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin`
 
-To begin using the wallet, click the "Connect Solo Wallet" button.
+![028](./assets/028.png)
 
-![agoric wallet solo connection](./assets/agoric-open-repl-1.png)
+Now that Docker has been installed you'll need to add your user account to the Docker group.
+`sudo usermod -aG docker $USER`
 
-After your solo wallet is connected, then you're ready to deploy the contract and API. 
+![029](./assets/029.png)
 
-## Deploy the Contract and API
+Now go ahead and reboot your image.
 
-In our second terminal, deploy the contract to the simulated blockchain
-and the API to the solo client.
+![030](./assets/030.png)
 
-```sh secondary style2
-# Terminal 2
-cd demo # if not already there
-agoric deploy ./contract/deploy.js 
-agoric deploy ./api/deploy.js
-```
+Once your image has rebooted and you've logged back on, test that Docker works by running the `hello-world` sample.
+`docker run hello-world`
 
-We'll cover [deploying smart contracts](./deploying.md)
-in detail later.
+![031](./assets/031.png)
 
-## Start the Dapp User Interface
 
-The web user interface communicates with the API in the solo client as well as the wallet.
+# Installing the Sample Dapp
+Now you'll use yarn to pull down the sample dapp. The sample dapp will be placed in a subfolder named `demo`.
+`yarn create @agoric/dapp --dapp-base https://github.com/agoric-labs/ --dapp-template dapp-game-places demo`
 
-```sh secondary style3
-# Terminal 3
-cd demo # if not already there
-cd ui && yarn start
-```
+![024](./assets/024.png)
+
+
+# Install Dapp Dependencies
+Now navigate to the `demo` directory and run the `yarn install` command to install any solution dependencies.
+`cd demo`
+`yarn install`
+
+![033](./assets/033.png)
+
+
+# Starting the Network
+Now go ahead and start the network using the `yarn start` command.
+`yarn start:docker`
+
+![034](./assets/034.png)
+
+Once the network has started you can check the logs. Once you see messages showing blocks with a status of `commit` you can rest assured the network is running properly.
+`yarn docker:logs`
+
+![035](./assets/035.png)
+
+
+# Starting the Dapp Smart Contract
+Exit the log window and start the smart contract by running the `yarn start` command.
+`yarn start:contract`
+
+![036](./assets/036.png)
+
+
+# Installing Keplr Wallet
+Next, you'll install the Keplr wallet plug-in. Open up your browser and navigate to [https://www.keplr.app/download](https://www.keplr.app/download). Select the version appropriate to your browser. 
+
+![037](./assets/037.png)
+
+Once the plug-in has been installed, open Keplr and select the option to "Import an existing wallet". Then choose the option to "Use recovery phrase or private key".
+
+![038](./assets/038.png)
 
-Leave this running in its own terminal window and visit [http://localhost:3000](http://localhost:3000) in a web browser.
+![040](./assets/040.png)
```

Comment:
These keplr screenshots are good, but they're kinda awkwardly large. I suggest using a width of 800 pixels.

```suggestion
<img width="800" src="./assets/040.png" />
```

It would be nice to have more meaningful filenames than `040.png`, but I'm not inspired with names either just now.

We really should have good alt text, though.

Reply:
^^ Noted